Dockerfile.s390x

```diff
@@ -14,4 +14,5 @@ COPY --from=base /opt/cni/bin /opt/cni/bin
 ADD bin/s390x/ /opt/cni/bin/
 
 ENV PATH=$PATH:/opt/cni/bin
-WORKDIR /opt/cni/bin]
\ No newline at end of file
+WORKDIR /opt/cni/bin]
```

Comment:
This is still broken see: https://github.com/projectcalico/cni-plugin/pull/904#issuecomment-661754566